ckcp/openshift_dev_setup.sh

```diff
@@ -8,19 +8,14 @@ SCRIPT_DIR="$(
   pwd
 )"
 
+source "$SCRIPT_DIR/hack/util/util-development-patch.sh"
+source "$SCRIPT_DIR/hack/util/util-setup-compute.sh"
+
 GITOPS_DIR="$(dirname "$SCRIPT_DIR")/gitops"
 CKCP_DIR="$(dirname "$SCRIPT_DIR")/ckcp"
+DEFAULTS="$(dirname "$SCRIPT_DIR")/ckcp/defaults.yaml"
```

Comment:
Should be `DEFAULTS="$CKCP_DIR/defaults.yaml"`

Reply:
done